src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserStateObjectFactory.Unix.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Data.SqlClient.SNI;
+
+namespace System.Data.SqlClient
+{
+    internal sealed class TdsParserStateObjectFactory
+    {
+
+        public static bool useManagedSni = true;
```

Comment:
Looks like this variable is only used in *.Windows.cs files. Is it still necessary here?